axelor-human-resource/src/main/java/com/axelor/apps/hr/service/expense/ExpenseService.java

```diff
@@ -18,74 +18,13 @@
  */
 package com.axelor.apps.hr.service.expense;
 
-import com.axelor.apps.account.db.Invoice;
-import com.axelor.apps.account.db.InvoiceLine;
-import com.axelor.apps.account.db.Move;
 import com.axelor.apps.base.AxelorException;
-import com.axelor.apps.base.db.BankDetails;
-import com.axelor.apps.base.db.Product;
 import com.axelor.apps.hr.db.Employee;
 import com.axelor.apps.hr.db.Expense;
-import com.axelor.apps.hr.db.ExpenseLine;
-import com.axelor.apps.hr.db.KilometricAllowParam;
-import com.axelor.message.db.Message;
-import java.io.IOException;
 import java.math.BigDecimal;
-import java.util.List;
-import wslite.json.JSONException;
 
 public interface ExpenseService {
```

Comment:
please continue to split what is left, maybe you can create `ExpenseToolService` for some methods, also `ExpenseComputationService` or even create a specific service for each public method left.